OTAnalytics/application/progress.py

```diff
@@ -23,25 +23,35 @@ def __init__(
         sequence: Sequence,
         counter: Counter,
         notify: Optional[Callable[[], None]] = None,
+        step_percentage: int = 1,
```

Comment:
Since `step_percentage` is of type int, I assume that the values are within in the bounds of [1,100]. Is that correct? I think it would be best to validate the value  in the constructor.

Reply:
Yes, I added a validation.